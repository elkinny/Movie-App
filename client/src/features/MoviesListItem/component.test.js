import MoviesListItemComponent from './component';
import React from 'react';
import { shallow } from 'enzyme';

describe('MoviesListItemComponent: ', () => {
  it('1. To match snapshot', () => {
    const props = {
      id: '1',
      runtime: '120min',
      vote_average: '4.0',
      title: 'Captain Marvel',
      genres: 'Action',
      release_date: 2019,
      overview:
        'Carol Danvers becomes one of the universe`s most powerful heroes when Earth is caught in the middle of a galactic war between two alien races.',
      link: 'https://www.youtube.com/watch?v=0LHxvxdRnYc',
      poster_path:
        'https://m.media-amazon.com/images/M/MV5BMTE0YWFmOTMtYTU2ZS00ZTIxLWE3OTEtYTNiYzBkZjViZThiXkEyXkFqcGdeQXVyODMzMzQ4OTI@._V1_SY1000_CR0,0,675,1000_AL_.jpg',
    };
    const component = shallow(<MoviesListItemComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
