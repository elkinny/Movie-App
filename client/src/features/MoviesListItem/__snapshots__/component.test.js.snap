// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesListItemComponent:  1. To match snapshot 1`] = `
<div
  className="movie-item movie-item--cy"
>
  <div
    className="movie-item__header"
  >
    <h2
      className="movie-item__title"
    >
       
      Captain Marvel
       
    </h2>
    <span
      className="movie-item__release_date"
    >
       
      2019
       
    </span>
  </div>
  <div
    className="movie-item__body"
  >
    <img
      className="movie-item__img"
      height="400"
      src="https://m.media-amazon.com/images/M/MV5BMTE0YWFmOTMtYTU2ZS00ZTIxLWE3OTEtYTNiYzBkZjViZThiXkEyXkFqcGdeQXVyODMzMzQ4OTI@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
    />
    <span
      className="movie-item__genres"
    >
       
      Action
       
    </span>
    <p
      className="movie-item__overview"
    >
      Carol Danvers becomes one of the universe\`s most powerful heroes when Earth is caught in the middle of a galactic war between two alien races.
    </p>
    <LinkButton
      cssClass="movie-item__link--cy"
      to="/1/#"
    >
      See more
    </LinkButton>
  </div>
</div>
`;
